IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/PropertySets/Pset_ImprovedGroundTypeGrouted/DocPropertySet.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_ImprovedGroundTypeGrouted" UniqueId="69f627a9-30cb-42a8-b0f5-92a0259ab9f8" ApplicableType="IfcImprovedGround/(.GROUTED.)">
```

Comment:
Hm, hm. Apparently you have an older version here that was taking into account selects of enums. We need to fix the applicability encoding to:
IfcImprovedGround/GROUTED

Reply:
Fixed in [78eaa5e](https://github.com/bSI-InfraRoom/IFC-Specification/pull/677/commits/78eaa5e1575b2014e8df32a1578333b14cf0ad96).